app/page.tsx

```
@@ -9,19 +9,32 @@ import {
     TrascendingCodeCard,
     WakatimeStatsCard,
 } from "@/components/organisms/";
+import {
+    AboutCard,
```

Biome (1.9.4)

[error] Shouldn't redeclare 'AboutCard'. Consider to delete it or rename it.

'AboutCard' is defined here:


(lint/suspicious/noRedeclare)